dapps/shop/backend/docs/api.md

```diff
@@ -0,0 +1,100 @@
+# API Endpoints
+
+## Authentication & Store Management
+
+Use the following for the test curl commands below:
+
+    export COOKIE_STORAGE="/tmp/test-cookies.txt"
+    export API_ROOT="http://127.0.0.1:3000"
+    export DEFAULT_OPTIONS="-vk"
+    export JSON_POST_OPTIONS="-v -X POST -H 'Content-Type: application/json'"
+    export JSON_DELETE_OPTIONS="-v -X DELETE -H 'Content-Type: application/json'"
+    export KEY_AUTH_HEADER="-H 'Authorization: bearer token123'"
+
+### `GET /auth`
+
+Check if user is authenticated
+
+    curl $DEFAULT_OPTIONS -b $COOKIE_STORAGE $API_ROOT/auth
+
+### `POST /auth/login`
+
+Authenticate a seller. Sessions tracked by cookies.
+
+    curl -v $JSON_POST_OPTIONS -d '{ "email": "test@example.com", "password": "thisismypassword" }' -c $COOKIE_STORAGE $API_ROOT/auth/login
+
+### `POST /auth/registration`
```

Comment:
Mike can we get another API method in here to determine if a user is registered? E.g, just POST their email and it'll return 201 if register or 404 if not. This is to handle the sign in or sign up flow in the admin thing.

Reply:
Will do.  We worried about potential PII leak here?  Maybe just add a TODO to add strict rps limits.

Reply:
added `/auth/:email`.  Still needs a store auth key.

Reply:
Yea people will be able to enumerate email addresses which is kind of annoying, probably an OK trade off though.